smartsim/_core/utils/helpers.py

```diff
@@ -273,3 +275,30 @@ def installed_redisai_backends(
     }
 
     return {backend for backend in backends if _installed(base_path, backend)}
+
+
+def get_ts() -> int:
+    """Helper function to ensure all timestamps are converted to integers"""
+    return int(datetime.timestamp(datetime.now()))
+
+
+def encode_cmd(cmd: t.List[str]) -> str:
```

Comment:
I encoded those CLI params to avoid a headache of having some unknown "user program" w/a weird argument cause some strange problem passing arguments through. With redis/colocated, we know what those arguments formats are and shouldn't see the same type of issue. 

I don't think we'll need to reuse this (and per our offline convo, we may choose to switch to "-- <extra args>" to avoid encoding altogether).